plugins/inlinetoolbar/skins/default.css

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+For licensing, see LICENSE.md or https://ckeditor.com/license
+*/
+
+.cke_balloon.cke_inlinetoolbar
+{
+	position: absolute;
+	z-index: 10000;
+	outline: none;
+}
+
+.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle
+{
+	position: absolute;
+	border-style: solid;
+	display: block;
+	width: 0;
+	height: 0;
+}
+
+.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle_inner,
+.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle_outer
+{
+	z-index: 0;
+}
+
+/* align: [ hcenter ] */
+.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle_outer.cke_balloon_triangle_align_hcenter
+{
+	left: 50%;
+	margin-left: -20px;
+}
+
+.cke_balloon.cke_inlinetoolbar
```

Comment:
This is the same selector as in line 6 (https://github.com/ckeditor/ckeditor-dev/pull/1136/files#diff-a222e1835c2e1c3b856fdaf98d45e2faR6), it should be merged into one rule.